ovs_container/ovs_container_start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/usr/share/openvswitch/scripts/ovs-ctl start
+ovs-vsctl set-manager ptcp:5678
+ovs-vsctl set Open_vSwitch . other_config:dpdk-extra="-w 0000:02:00.1,representor=[0-3],dv_flow_en=0,dv_esw_en=0,isolated_mode=1"
```

Comment:
pci and representor=[0-3] should be passed as args it can be several (like in the bond case)

Reply:
Done